src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -5,49 +5,49 @@
       "Size": 3032
     },
     "assemblies/Java.Interop.dll": {
-      "Size": 59377
+      "Size": 59420
```

Comment:
Maybe don't actually update these files here, as they are getting updated in:

https://github.com/xamarin/xamarin-android/pull/7162#issuecomment-1184462655